src/extension/wps/src/main/java/au/org/emii/gogoduck/worker/FeatureSourceIndexReader.java

```diff
@@ -29,21 +29,34 @@ public FeatureSourceIndexReader(UserLog userLog, Catalog catalog) {
 
     @Override
     public URIList getUriList(String profile, String timeField, String urlField, GoGoDuckSubsetParameters subset) throws GoGoDuckException {
-        String timeCoverageStart = subset.get("TIME").start;
-        String timeCoverageEnd = subset.get("TIME").end;
 
         URIList uriList = new URIList();
 
+        // Setting uri list for config GoGoDuck modules
```

Comment:
I don't think we need to support (and I don't want to support) setting a file location in the config.

CARS has been harvested to the database and we should go forward with that.  TPAC also don't required this.